unit_tests/test_dbprocessing.py

```diff
@@ -25,23 +27,37 @@ class ProcessQueueTestsBase(unittest.TestCase, dbp_testing.AddtoDBMixin):
     def setUp(self):
         """Make a db and open it so have something to work with"""
         super(ProcessQueueTestsBase, self).setUp()
-        self.td = tempfile.mkdtemp()
-        shutil.copy2(
-            os.path.join(os.path.dirname(__file__), 'emptyDB.sqlite'),
-            self.td)
+
+        if 'PGDATABASE' in os.environ.keys():
+            self.td = os.environ['PGDATABASE']
+        else:
+            self.td = tempfile.mkdtemp()
+            shutil.copy2(
+                os.path.join(os.path.dirname(__file__), 'emptyDB.sqlite'),
+                self.td)
+
         # Set up the baseline mission environment, BEFORE making processqueue
         self.addSkeletonMission()
-        self.pq = dbprocessing.dbprocessing.ProcessQueue(
-            os.path.join(self.td, 'emptyDB.sqlite'))
+        if 'PGDATABASE' in os.environ.keys():
+            self.pq = dbprocessing.dbprocessing.ProcessQueue(self.td)
+        else:
+            self.pq = dbprocessing.dbprocessing.ProcessQueue(
+                    os.path.join(self.td, 'emptyDB.sqlite'))
```

Comment:
We should probably get all the postgres vs. sqlite testing stuff handling once in dbp_testing and make the other scripts ignorant of it.

Reply:
I agree.    I just wrap you sql code where I found it.

Reply:
Patched up a bit for now; the full pull-out will come later.